### PR TITLE
Add NAP1: institutional and funding partners

### PR DESCRIPTION
**Pull request by jni**
# Description

This PR creates a new [napari Advancement Proposal](https://napari.org/naps) :sleeping: proposing changes to our governance to explicitly delineate how companies and institutions can support napari, be it with funding or with employee time. It explains what benefits funders of the project can and cannot expect from their support of napari.

Please read the NAP for further details!

I'd like to propose also that the NAP be merged (note *merged*, not accepted) as soon as possible, as detailed in the [NAP process](https://napari.org/naps/0-nap-process.html#nap-workflow):

> At the earliest convenience, the PR should be merged (regardless of whether it is accepted during discussion). A NAP that outlines a coherent argument and that is considered reasonably complete should be merged optimistically, regardless of whether it is accepted during discussion. Additional PRs may be made by the author to update or expand the NAP, or by maintainers to set its status, discussion URL, etc.

We can continue to make modifications over the coming week, and my hope is that we can finalise the wording and accept the NAP at the core developer gathering in San Francisco next weekend! :tada: But it would be really nice to have the NAP on the webpage with the coming release candidate!

### CODE REVIEW

**Comment by brisvag** [comment]
Yay, NAP1! :tada:

I'm still a but unclear on the procedure:
- should we let this sit here for some time for visibility?
- In case someone would like to propose changes, should it be done here, or preferably later in a separate PR?
- Or should the discussion happen elsewhere and then it's up to the author to update accordingly?

**Review by sofroniewn — APPROVED** [review]
Thanks for getting this going @jni.

As to process - if people want to weigh in here and do a round of feedback directly on the PR, I think that would be ok. Might as well get some input now, and if there are easy things the NAP authors want to fix/ change based on that feedback, might as well - or if there are more substantial things and we add another NAP author now that could be ok too. We could request review from the core dev team on this PR to start that feedback process?

If @brisvag you want to start a discussion in the zulip too that's fine also. I'm quite open - this is our first one!! @jni recommendations?

File: docs/naps/1-institutional-funding-partners.md
Comment (DragaDoncila) [inline docs/naps/1-institutional-funding-partners.md]:
```suggestion
- create an Institutional and Funding Partner Advisory Council (IFPAC), made
```

**Review by DragaDoncila — APPROVED** [review]
This seems excellent to me. Suggested a super tiny typo fix.

**Comment by sofroniewn** [comment]
My thoughts here are that I will merge this tomorrow as a record of the NAP and then open up a new PR with the proposed text. There can then be more detailed discussion on the proposed text on the PR or in zulip. Thanks!!

**Comment by jni** [comment]
Hi @brisvag and everyone!

Good Q about the process. As noted, NAPs should be merged with draft status as soon as is feasible. Comments can continue (a) on this PR even after merging, (b) on the [Zulip thread](https://napari.zulipchat.com/#narrow/stream/322105-naps/topic/nap-1!/near/280631529), (c) on new Zulip threads but please link here or in subsequent NAP1 PRs. Then it is the responsibility of the NAP authors (@sofroniewn and me) to make sure to summarise all discussion in the NAP in subsequent PRs.

@sofroniewn I think the PR *actually* adding the text to the governance should come with the proposal for NAP acceptance. Since most core devs will be in SFO when we actually make these changes, I think it's fine to make that PR/proposal (remember to make a Zulip thread!) immediately after merging this, and then we just need to make sure any concerns raised in the in-person meeting are properly recorded. 🎉

**Pull request merged**
